Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.webpushd.sb.in

```diff
@@ -117,6 +117,10 @@
     (literal "/Library/Preferences/com.apple.security.plist")
     (home-literal "/Library/Preferences/com.apple.security.plist"))
 
+(allow file-read*
+    (literal "/System/Library/DarwinDirectory/system/recordStore.data")
+    (literal "/private/var/db/DarwinDirectory/local/recordStore.data"))
```

Comment:
Are these needed for push on macOS?

Reply:
I believe this is just iOS, currently.

Thanks for reviewing!